src/bp/ui-studio/src/web/components/Layout/StatusBar/TrainingStatus/SingleLang.tsx

```diff
@@ -85,22 +86,24 @@ const TrainingStatusComponent: FC<Props> = (props: Props) => {
           {message}
         </span>
 
-        {status === 'needs-training' && (
-          <Button minimal className={style.button} onClick={onTrainClicked} disabled={loading}>
-            {lang.tr('statusBar.trainChatbot')}
-          </Button>
-        )}
         {status === 'training-pending' && (
```

Comment:
What's up with training pending ? This means if a use who has access queues a training, for a split second, the studio will display a spinning wheel to the use who doesnt have access ?

Reply:
Ok, the content-editor can see that a training is occuring, just cannot click train or cancel :+1: